com.beeper.Beeper.yaml

```diff
@@ -0,0 +1,90 @@
+#This package is not affiliated to Beeper Team
+app-id: com.beeper.Beeper
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: startbeeper
+rename-icon: beeper
+separate-locales: false
+finish-args:
+  # Version required to use the document-portal v4
+  - --require-version=1.7.0
+  # Required due to being a GUI application
+  - --socket=x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required to provide Call functionality
+  - --socket=pulseaudio
+  - --device=all
+  # As a chat application, networking is required
+  - --share=network
+  # Required for appindicator in various desktop environments
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required to allow screensaver/idle inhibition such as during video calls
+  - --talk-name=org.freedesktop.ScreenSaver
+  # Required for advanced input methods e.g. writing CJK languages
+  - --talk-name=org.freedesktop.portal.Fcitx
+  # Required to store access tokens (persistent logins)
+  - --filesystem=xdg-run/keyring
+  # Required for experimental wayland support
+  - --filesystem=xdg-run/pipewire-0
+  # For correct cursor scaling under Wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /share/pkgconfig
+  - /share/aclocal
+  - /share/gtk-doc
+  - /share/doc
+  - /share/info
+  - /share/man
+  - /man
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: beeper
+    buildsystem: simple
+    build-commands:
+      - chmod +x Beeper.AppImage
+      - ./Beeper.AppImage --appimage-extract
+
+      - desktop-file-edit --set-icon ${FLATPAK_ID} --set-key Exec --set-value 'startbeeper
+        %u' squashfs-root/beeper.desktop
+      - install -D squashfs-root/beeper.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -D startbeeper /app/bin/
+      - install -D com.beeper.Beeper.metainfo.xml /app/share/metainfo/com.beeper.Beeper.metainfo.xml
+      - mkdir -p /app/share/icons/hicolor
+      - cp -r squashfs-root/usr/share/icons/hicolor/* ${FLATPAK_DEST}/share/icons/hicolor
+      # flatpak doesn't support icons larger than 512x512
```

Comment:
to be pedantic, it's not flatpak. It's the XDG icon specification. Flatpak just is more strict enforcing rules.

Reply:
Changed